l10n_be_cooperator_national_number/models/res_partner_id_category.py

```diff
@@ -0,0 +1,15 @@
+# SPDX-FileCopyrightText: 2024 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from odoo import api, models
+
+
+class ResPartnerIdCategory(models.Model):
+    _inherit = "res.partner.id_category"
+
+    @api.model
+    def get_belgian_id_category(self):
```

Comment:
please keep the name that was used in `subscription.request` (`get_be_national_register_number_id_category()`) since there is another category for the identity card number (`l10n_be_id_card_category`), and the current name is confusing.